AlfrescoContent/AlfrescoContent/Classes/Swaggers/Process/StartFormFields.swift

```diff
@@ -496,6 +508,90 @@ public enum ValueUnion: Codable {
             return num
         case .null:
             return nil
+        case .int(_):
+            return nil
+        case .bool(_):
+            return nil
+        case .valueElementDict(_):
+            return nil
+        case .assignee(_):
+            return nil
+        }
+    }
+    
+    public func getIntValue() -> Int? {
+        switch self {
+        case .string(_):
+            return nil
```

Comment:
If you are returning the nil in multiple cases then use else instead of handle on every case.

Eg:
public func getBoolValue() -> Bool? {
    switch self {
    case .bool(let value):
        return value
    default:
        return nil
    }
}